controller/admin-controller.js

```diff
@@ -0,0 +1,118 @@
+const Product = require("../model/product");
+const { success, error } = require("../response-api/responseApi");
+const { statusCode, message } = require("../constant/constant");
+
+// ? Input : Title , Price, Description --- Output : Add Product to the Product Database
+
+const addProduct = async (req, res) => {
+  try {
+    const { title, price, description } = req.body;
+    const product = new Product({
+      title,
+      price,
+      description,
+    });
+    await product.save();
+    res
+      .status(statusCode.CREATED)
+      .json(success("success", message.ADD_PRODUCT, res.statusCode));
+  } catch (err) {
+    res
+      .status(statusCode.INTERNAL_SERVER_ERROR)
+      .json(error(message.SERVER_ERROR, res.statusCode));
+  }
+};
+
+// ? Output : Page with all products
+
+const getProductList = async (req, res) => {
+  try {
+    const products = await Product.find();
```

Comment:
where is pagination ?

Reply:
At the time of generating this PR, I was not working on Pagination so it is not added in this particular commit.
It will be there in the next PR.